### PR TITLE
Quit button and result page

### PR DESCRIPTION
**Pull request by na50r**
I added a Quit button that allows you to quit mid-game and also fixed up the result page for the loser screen.

https://www.youtube.com/watch?v=umz3iPWzCCU

### CODE REVIEW

File: src/components/views/Result.tsx [hunk 1]
```diff
@@ -38,6 +40,7 @@ const Result = () => {
             try {
                 let response = await api.get(`/lobbies/${lobbyCode}/players`,);
                 setPlayers(response.data.map(p => new Player(p)));
+                console.log(players)
```
Comment (Tmmn) [inline src/components/views/Result.tsx:43]: do you still need this?

**Review by Tmmn — APPROVED** [review]
could not find any issues. Played some games and could always quit and result screen was displayed correctly

File: src/components/views/Result.tsx (hunk 1 above)
Comment (na50r) [reply src/components/views/Result.tsx:43]: Nope

File: src/components/views/GameBoard/Game.tsx [hunk 2]
```diff
@@ -86,6 +86,24 @@ const Game = ({ stompWebSocketHook }) => {
...
+        navigate("/lobbyoverview");
+    };
+
+    useEffect(() => {
+        let messagesLength = stompWebSocketHook.messages.length;
+        if (messagesLength > 0 && stompWebSocketHook.messages[messagesLength - 1] !== undefined) {
+            const newObject = stompWebSocketHook.messages[messagesLength - 1];
+            if (newObject.instruction === "kick") {
+                kick();
+            }
```
Comment (Tmmn) [inline src/components/views/GameBoard/Game.tsx:102]: move this block between line 85 and 86 and remove this effect

**Review by Tmmn — APPROVED** [review]

**Comment by Tmmn** [comment]
don't forget to add bug issues and reference them

File: src/components/views/GameBoard/Game.tsx
Comment (Tmmn) [inline src/components/views/GameBoard/Game.tsx:1]: 👍🏼

**Pull request merged**